lib/vrl/web-playground/Cargo.toml

```diff
@@ -121,6 +121,9 @@ features = [
     "parse_user_agent",
     "parse_xml",
     "push",
+    "random_bool",
```

Comment:
Thanks for remembering to add this! Hopefully this will be better enforced in the future.